Jenkinsfile

```diff
@@ -6,7 +6,7 @@ pipeline {
   agent any
   environment {
     REPO = 'apm-agent-dotnet'
-    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    BASE_DIR = "src"
```

Comment:
we've reached the max path length 248 characters because of that NuGet 5.1.0 fails when makes the restore.